packages/shared/src/utils/storage/storage.ts

```diff
@@ -36,51 +45,56 @@ export const isStorageSupported = storage => {
     }
 };
 
-const Store = function (storage) {
+const Store = function (this: { storage: Storage }, storage: Storage) {
     this.storage = storage;
     this.storage.getObject = getObject;
     this.storage.setObject = setObject;
 };
 
 Store.prototype = {
-    get(key) {
+    get(key: string) {
         return this.storage.getItem(key) || undefined;
     },
-    set(key, value) {
+    set(key: string, value: string) {
         if (typeof value !== 'undefined') {
             this.storage.setItem(key, value);
         }
     },
-    getObject(key) {
+    getObject(key: string) {
         return typeof this.storage.getObject === 'function' // Prevent runtime error in IE
             ? this.storage.getObject(key)
             : JSON.parse(this.storage.getItem(key) || '{}');
     },
-    setObject(key, value) {
+    setObject(key: string, value: unknown) {
         if (typeof this.storage.setObject === 'function') {
             // Prevent runtime error in IE
             this.storage.setObject(key, value);
         } else {
             this.storage.setItem(key, JSON.stringify(value));
         }
     },
-    remove(key) {
+    remove(key: string) {
         this.storage.removeItem(key);
     },
     clear() {
         this.storage.clear();
     },
 };
 
-const InScriptStore = function (object) {
+const InScriptStore = function (this: { store: unknown }, object?: unknown) {
     this.store = typeof object !== 'undefined' ? object : {};
 };
 
 InScriptStore.prototype = {
-    get(key) {
+    get(key: string) {
         return getPropertyValue(this.store, key);
     },
-    set(k, value, obj = this.store) {
+    set(
+        this: { store: any; set: (key: string | string[], value: string, obj: string[]) => void },
```

Comment:
```suggestion
        this: { store: unknown; set: (key: string | string[], value: string, obj: string[]) => void },
```

Reply:
unknown is not working here! because of the obj!